RuntimeUnityEditor/Windows/ObjectTree/RootGameObjectSearcher.cs

```diff
@@ -99,12 +101,14 @@ public void Search(string searchString, bool searchProperties, bool refreshObjec
 
                 _lastSearchString = searchString;
 
-                RuntimeUnityEditorCore.Logger.Log(LogLevel.Info, searchProperties ? $"Deep searching for [{searchString}], this can take a while..." : $"Searching for [{searchString}]");
+                RuntimeUnityEditorCore.Logger.Log(LogLevel.Info, searchProperties ? $"Deep searching for [{searchString}], this can take a while..." :
+                                                                 searchComponents ? $"Searching components for [{searchString}]" :
+                                                                                    $"Searching for [{searchString}]");
                 var sw = Stopwatch.StartNew();
 
                 _searchResults = GetRootObjects()
                     .SelectMany(x => x.GetComponentsInChildren<Transform>(true))
-                    .Where(x => x.name.Contains(searchString, StringComparison.InvariantCultureIgnoreCase) ||
+                    .Where(x => x.name.Contains(searchString, StringComparison.InvariantCultureIgnoreCase) || searchComponents &&
```

Comment:
Just added searchComponents here as a binary and to skip the component part for the new name only search